org.claws_mail.Claws-Mail.json

```diff
@@ -0,0 +1,161 @@
+          "url": "https://github.com/Yeraze/ytnef/archive/v1.9.3.tar.gz",
+          "sha256": "41a0033bde33c86a7e4aa4e14bb822dd03084098638e7d6557263e47e80b4f4f"
+        }
+      ]
+    },
+    {
+      "name": "libopenjpeg",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/uclouvain/openjpeg/archive/v2.3.1.tar.gz",
+          "sha256": "63f5a4713ecafc86de51bfad89cc07bb788e9bba24ebbf0c4ca637621aadb6a9"
+        }
+      ]
+    },
+    {
+      "name": "libpoppler-glib",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.freedesktop.org/poppler/poppler/-/archive/poppler-0.80.0/poppler-poppler-0.80.0.tar.gz",
+          "sha256": "e517d9e00394acf0ce4f4859eaf942e453e7e8c3c04c894df174be7e3baf16cb"
+        }
+      ]
+    },
+    {
+      "name": "libical",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/libical/libical/archive/v3.0.5.tar.gz",
+          "sha256": "483acbf7fee66ca071c2ff8183e46b6f2b3a89e1e866eadf4870eaaa281c8db1"
+        }
+      ]
+    },
+    {
+      "name": "libgumbo",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/google/gumbo-parser/archive/v0.10.1.tar.gz",
+          "sha256": "28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad"
+        }
+      ]
+    },
+    {
+      "name": "liboauth",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/x42/liboauth/archive/186fb14843c1bc98b70ece02c74b5696b3de9ce7.zip",
+          "sha256": "5d7a197cf56a7964efc37814b0121806972898e0cdf4fddab6e74249acf6fc83"
+        }
+      ],
+      "build-commands": [
+        "autoreconf --install",
+        "./configure --prefix=/app",
+        "make",
+        "make install"
+      ]
+    },
+    {
+      "name": "libuhttpmock",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.com/uhttpmock/uhttpmock/-/archive/cda4b6330c6543c48600dd015af44bd0c2a21b4d/uhttpmock-cda4b6330c6543c48600dd015af44bd0c2a21b4d.tar.gz",
+          "sha256": "13d3d471b366f7df123a3e85dc59491ce5d1f58ed2a4365cb84ad639d3f11e94"
+        }
+      ]
+    },
+    {
+      "name": "libgdata",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dgoa=disabled"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.gnome.org/GNOME/libgdata/-/archive/0.17.10/libgdata-0.17.10.tar.gz",
+          "sha256": "3f349002c078485ff5e9654a49fb382cac8290e50294eadd5349389166f57b91"
+        }
+      ]
+    },
+    {
+      "name": "bogofilter",
+      "buildsystem": "simple",
+      "build-commands": [
```

Comment:
Same as above.

Reply:
Will need to check how I can make this work, given that I need to go into a subdirectory first.

Reply:
Done.